router.py

```diff
@@ -309,10 +309,16 @@ async def update_device_trackers(self) -> None:
             device.update(dev_info, consider_home)
 
         for device_mac, dev_info in wrt_devices.items():
+            # Skip if we've already have this device
             if device_mac in self._devices:
                 continue
-            if not dev_info["name"]:
+
+            alias = dev_info.get("alias").strip()
+            name = dev_info.get("name").strip()
```

Comment:
```suggestion
            alias = dev_info.get("alias","").strip()
            name = dev_info.get("name","").strip()
```
you were right, i was wrong